generator/raw_out.py

```diff
@@ -0,0 +1,33 @@
+import re
+from collections import defaultdict
+from typing import DefaultDict
+from typing import List
+
+import requests
+
+TERM_URL = 'https://raw.githubusercontent.com/togakangaroo/tech-terms/master/terms.org'
+
+
+def convert_to_dict_list(items: List[dict]) -> DefaultDict[str, list]:
+    terms: DefaultDict[str, list] = defaultdict(list)
+
+    for item in items:
+        terms[item['term']].append(item['definition'])
+
+    return terms
+
+
+def parse_ank_raw_from_source() -> DefaultDict[str, list]:
+    r = requests.get(TERM_URL)
+    content = r.content
+
+    # this regex matches 
+    two_col_org_row = re.compile('^\\s*\\|\\s*(?P<term>.*?)\\s*\\|\\s*(?P<definition>.*?)\\s*\\|\\s*$')
+    lines = content.splitlines()
```

Comment:
So this is missing the part of my script where it made sure that there was exactly a term and a definition. If we want to add some header text to the table I wouldn't want it to break this.

Also maybe refactor the existing regex so its not just duplicated everywhere?